Layout.go

```diff
@@ -77,6 +77,21 @@ func EndGroup() {
 	C.iggEndGroup()
 }
 
+// BeginDisabled starts a disabled group.
```

Comment:
The `Disabled` API is in BETA status. I'd like to avoid adding more of those - every time they fundamentally change, this would need a new major version.